src/main/java/seedu/address/logic/Messages.java

```diff
@@ -57,14 +54,10 @@ public static String format(Person person) {
                 .append(person.getAddress())
                 .append(";\n")
                 .append("Appointment: ")
-                .append(person.getAppointment().map(Appointment::toString).orElse("N/A"))
```

Comment:
I liked the `.map()` change, it improved code quality a lot. May I suggest that you keep the `null` but use `map`?

Example:
```
person.getAppointment().map(Appointment::toString).orElse(null)
```

Reply:
Good suggestion, I will keep the use of map.